app/import_functions_test.go

```diff
@@ -2501,9 +2501,33 @@ func TestImportimportMultiplePostLines(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, 0, errLine)
 
+	// Create a pinned message.
+	data = LineImportWorkerData{
+		LineImportData{
+			Post: &PostImportData{
+				Team:     &teamName,
+				Channel:  &channelName,
+				User:     &user2.Username,
+				Message:  ptrStr("Pinned Message"),
+				CreateAt: ptrInt64(model.GetMillis()),
+				IsPinned: ptrBool(true),
+			},
+		},
+		1,
+	}
+	errLine, err = th.App.importMultiplePostLines(th.Context, []LineImportWorkerData{data}, false)
+	assert.Nil(t, err)
+	assert.Equal(t, 0, errLine)
+
+	resultPosts, nErr := th.App.Srv().Store.Post().GetPostsCreatedAt(channel.Id, *data.Post.CreateAt)
+	assert.NoError(t, nErr, "Expected success.")
+	resultPost := resultPosts[0]
+	pinnedStatus := resultPost.IsPinned
+	require.True(t, pinnedStatus, "This post should be pinned.")
```

Comment:
we can do this inline:
```suggestion
	require.True(t, resultPost.IsPinned, "This post should be pinned.")
```